modules/base/rendering/pointcloud/renderablepointcloud.cpp

```diff
@@ -207,6 +234,13 @@ namespace {
         openspace::properties::Property::Visibility::User
     };
 
+    constexpr openspace::properties::Property::PropertyInfo HasOrientationDataInfo = {
+        "HasOrientationData",
+        "Has Orientation Data",
+        "Set to true if orientation data was read from the dataset",
```

Comment:
```suggestion
        "Set to 'true' if orientation data was read from the dataset",
```

Reply:
This description doesn't really seem actionable.  Why would I want to change it at runtime as a user?

Reply:
Nevermind, just saw that it was read-only.  Do we need it as a property or could it just be a plain `bool` as well?

Reply:
The reason I have it as a property is to provide information about it in the user interface for the user

Otherwise, you have to go and look in the datafile itself to know whether it has any orientation data that could be used.  And I don't think we want the users to do this

Reply:
Also, we do not have these quotation marks in any other property descriptions when talking about bool properties. So I don't agree with that we should add it here

Reply:
Mixed up the Doxygen, where we have `true` with the property descriptions

Reply:
Nevermind.. I take it back (but still fine with not having them here, to be clear).  We should have a decision about which one we prefer, since we do use '..' in other places:

- https://github.com/OpenSpace/OpenSpace/blob/master/modules/base/rendering/renderablesphereimagelocal.cpp#L50
- https://github.com/OpenSpace/OpenSpace/blob/master/modules/base/rendering/renderabletrail.cpp#L83-L84
- https://github.com/OpenSpace/OpenSpace/blob/master/modules/base/rotation/fixedrotation.cpp#L43-L44
- https://github.com/OpenSpace/OpenSpace/blob/master/modules/globebrowsing/src/globetranslation.cpp#L80
- ...

Found 81 matches for `'true'`  many of them in property descriptions.   I'll put a "Discuss documentation formatting" on the standing list